crates/typos-cli/Cargo.toml

```diff
@@ -77,6 +77,7 @@ colorchoice-clap = "1.0.3"
 serde_regex = "1.1.0"
 regex = "1.10.4"
 encoding_rs = "0.8.34"
+serde-sarif = "0.4.2"
```

Comment:
The docs have this warning:

> [WARNING: VERY UNSTABLE (EARLY IMPLEMENTATION)](https://docs.rs/serde-sarif/latest/serde_sarif/#warning-very-unstable-early-implementation)

Reply:
https://github.com/psastras/sarif-rs/commit/9b4adbe37359bbc11a338e4e1d789d81f26ce535

This was added three years ago. As I only used the generated code from a json schema, I think this won't be a problem.

Reply:
The warning is still there and I would like more concrete detail (and not supposition) on why its there.

Reply:
Then maybe an issue confirmation should be opened in the upstream repository?

Reply:
https://github.com/psastras/sarif-rs/issues/646

But the author doesn't seem to be active and also takes a long time to respond.

Reply:
The author set it to stable now.

Reply:
Ok